app/web/src/newhotness/Explore.vue

```diff
@@ -1,78 +1,118 @@
 <template>
-  <section class="grid gap-md h-[100%]">
-    <div>
-      <!-- filter / top header -->
-      <InstructiveVormInput
-        classes="py-2"
-        :activeClasses="
-          clsx(themeClasses('border-action-500', 'border-action-300'))
+  <section class="grid h-full">
```

Comment:
i don't believe `h-full` is needed, thats a default with grid as far as I can tell?